src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -29,4 +33,26 @@
     }
     return generatedImports;
   }
+
+  public List<ImportTypeView> generateImports(SurfaceTransformerContext context) {
```

Comment:
This is meant to be permanent. The imports generated in this function are for languages that import files based on the proto file name as opposed to being imported based on the fully qualified typename like in Java/PHP. I'll change the naming to make this point a bit more clear.